Documentation/writing-tizen-csharp-apps.md

```diff
@@ -0,0 +1,131 @@
+# Writing a Tizen .NET Application
+
+This document provides you how to write a Tizen .NET Application with Machine Learning APIs.
+Since Tizen 5.5 (not released yet), Machine Learning Inference functionality has been provided on Mobile, Wearable and TV profile.
```

Comment:
Remove ```(not released yet)```. Please assume that it's already released. We don't want to update docs with Tizen releases.

Reply:
ACK!